GRIB2_Template_3_150_GridDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,14 @@
+Title_en,OctetNo,Contents_en,Note_en,noteIDs,codeTable,flagTable,Status
+"Hierarchical Equal Area isoLatitude Pixelization grid (HEALPix)",15,Shape of the Earth,(see Code table 3.2),,3.2,,Operational
+"Hierarchical Equal Area isoLatitude Pixelization grid (HEALPix)",16,Scale factor of radius of spherical Earth,,,,,Operational
+"Hierarchical Equal Area isoLatitude Pixelization grid (HEALPix)",17-20,Scaled value of radius of spherical Earth,,,,,Operational
+"Hierarchical Equal Area isoLatitude Pixelization grid (HEALPix)",21,Scale factor of major axis of oblate spheroid Earth,,,,,Operational
```

Comment:
@sebvi octets 21-30 differ from the table in the issue. 

e.g. "Scale factor of Earth major axis" != "Scale factor of major axis of oblate spheroid Earth"

Reply:
I have use for the template the same wording than in all other section 3 template. I could update the issue description to reflect that if really necessary.